lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -1,6 +1,6 @@
 # These dependencies are only required when certain config options are set
 psycopg2==2.6.1
-WebError==0.11
+WebError==0.10.3
```

Comment:
Hrmm.  I just updated what was there, but you're on to something here.  I think it should probably be removed from the *other* requirements file, though.